pageserver/src/consumption_metrics.rs

```diff
@@ -19,6 +19,9 @@ use tracing::*;
 use utils::id::{NodeId, TenantId, TimelineId};
 
 const WRITTEN_SIZE: &str = "written_size";
+/// Values will be the difference of the latest written_size (last_record_lsn) to what we
+/// previously sent.
+const WRITTEN_SIZE_DELTA: &str = "written_size_delta";
```

Comment:
you can add it also to https://github.com/neondatabase/neon/blob/b98419ee56891ce99f6da808ea2aaece298e8b5a/test_runner/regress/test_metric_collection.py#L33

so it would be verified in the e2e tests

Reply:
It will not be always available so testing with this case is not as straighforward...